orttraining/orttraining/core/graph/optimizer_config.h

```diff
@@ -6,7 +6,11 @@
 #include <string>
 #include <unordered_map>
 #include "core/common/logging/logging.h"
+#include "core/framework/tensor.h"
```

Comment:
nit: can these be in alphabetical order?

Reply:
why were these includes added to this header? the only other new code i see is some NameMLValMap data members.

---
In reply to: [533844188](https://github.com/microsoft/onnxruntime/pull/5869#discussion_r533844188) [](ancestors = 533844188)

Reply:
two of them are required to get it to compile, removed two other redundant includes